localization/i18n/pl/OrcaSlicer_pl.po

```diff
@@ -11863,52 +11862,63 @@ msgstr "Głębokość zazębiania się podzielonego na segmenty obszaru"
 
 msgid "Interlocking depth of a segmented region. Zero disables this feature."
 msgstr ""
-"Głębokość blokowania obszaru segmentowego. Wartość zero wyłącza tę funkcję."
+"Głębokość zazębiania się podzielonego na segmenty regionu. Wartość zero "
+"wyłącza tę funkcję."
 
 msgid "Use beam interlocking"
-msgstr ""
+msgstr "Użyj struktury zatrzaskowej"
 
 msgid ""
 "Generate interlocking beam structure at the locations where different "
 "filaments touch. This improves the adhesion between filaments, especially "
 "models printed in different materials."
 msgstr ""
+"Wygeneruj strukturę zazębiającą się w miejscach, gdzie stykają się różne "
+"filamenty. Poprawia to przyczepność między filamentami, szczególnie w "
+"modelach drukowanych z różnych materiałów."
 
 msgid "Interlocking beam width"
-msgstr ""
+msgstr "Szerokość struktury zatrzaskowej"
 
 msgid "The width of the interlocking structure beams."
-msgstr ""
+msgstr "Określa szerokość struktury zatrzaskowej"
 
 msgid "Interlocking direction"
-msgstr ""
+msgstr "Kierunek zatrzasku"
 
 msgid "Orientation of interlock beams."
-msgstr ""
+msgstr "Orientacja struktury zatrzaskowej"
 
 msgid "Interlocking beam layers"
-msgstr ""
+msgstr "Liczba warstw struktury zatrzaskowej"
 
 msgid ""
 "The height of the beams of the interlocking structure, measured in number of "
 "layers. Less layers is stronger, but more prone to defects."
 msgstr ""
+"Wysokość struktury zatrzaskowej wyrażona w liczbie warstw. Mniejsza liczba "
+"warstw oznacza większą wytrzymałość, ale większą podatność na wady."
 
 msgid "Interlocking depth"
-msgstr ""
+msgstr "Głębokość zazębiania"
```

Comment:
Before (and after) you've used "struktury zatrzaskowej" instead of "zazębiania". Personally I think "zazębianie" is better term since "struktura zatrzaskowa" isn't "zatrzaskowa" but "zazębiająca" ;)

Reply:
Tak ma to sens, poprawie.